src/components/grid-list/grid-list.scss

```diff
@@ -64,4 +75,32 @@ md-grid-tile {
   md-grid-tile-footer {
     bottom: 0;
   }
+
+  [md-line] {
+    display: block;
+    overflow-x: hidden;
+    white-space: nowrap;
+    text-overflow: ellipsis;
+    margin: 0;
+    padding: 0;
+    font-weight: normal;
+    font-size: $md-grid-font-size;
+
+    &:nth-child(2) {
```

Comment:
The `:nth-child(2)` seems somewhat opaque; add a comment explaining what it's for?